crowbar_framework/app/models/service_object.rb

```diff
@@ -1576,4 +1616,26 @@ def profile(name, &block)
       block.call
     end
   end
+
+  def equal_or_match(value, maybe_regexp)
+    operator = lambda do
+      op, check = maybe_regexp.split(" ", 2)
+      value.to_f.send(op.to_sym, check.to_f)
```

Comment:
Well, I'm also worried about 12.04 vs 12.10 comparisons (which I think is something you raised earlier). Isn't it just broken right now?